sql/5_0_2-to-5_0_3_upgrade.sql

```diff
@@ -653,3 +653,11 @@ CREATE UNIQUE INDEX `token` ON `api_token` (`token`);
 #IfNotIndex patient_access_onsite pid
 CREATE UNIQUE INDEX `pid` ON `patient_access_onsite` (`pid`);
 #EndIf
+
+#IfMissingColumn form_encounter uuid
+ALTER TABLE `form_encounter` ADD `uuid` binary(16) NOT NULL default '';
+#EndIf
+
```

Comment:
This code block will go into the middle of these 2 code blocks:
```
#IfUuidNeedUpdate form_encounter
#EndIf
```
This will populate the all the empty uuid for users that are upgrading from a prior OpenEMR versions. So, when making the uuid for upgrade, there are 3 steps:
1. Create the uuid column (you did above)
2. Populate the blank uuids with a uuid (with code I copied here)
3. Then make the uuid column a unique index (you did below)

Note that if you skip step 2, then it will break on step 3, since blank columns are not unique.

Reply:
Actually I was unaware of how Comments Meta language is working but now after going through this [file](https://github.com/openemr/openemr/blob/81eaedc5377f46590445367e696ffdcc60da71de/library/sql_upgrade_fx.php ). I have a little understanding of how things are wired.

Thanks